frontend/lib/src/components/core/Block/Block.tsx

```diff
@@ -302,6 +302,7 @@ const VerticalBlock = (props: BlockPropsWithoutWidth): ReactElement => {
 
   useEffect(() => {
     if (wrapperElement.current) {
+      setWidth(wrapperElement.current.getBoundingClientRect().width)
```

Comment:
I am a little bit worried that this might have side-effects on other scenarios since it is on the generic `Block.tsx` element. Would it make sense and be possible to move this into the map component, since the issue seem to appear only for maps, right?

Reply:
Sure, I can look into moving it into just maps!

Reply:
@sfc-gh-nbellante I have just had another look and in general this change here should be fine. But I also played around with your solution and it looked to me like this line here is not even needed. I commented out and both issues seemed to be fixed with just the changes in `DeckGlJsonChart.tsx`. Are you able to confirm this or am I doing something wrong? 🙂

Reply:
Chatted in DMs, we don't know why, but this seems to no longer be needed. 👻